src/main/frontend/pipeline-console-view/pipeline-console/main/hooks/use-steps-poller.ts

```diff
@@ -11,6 +11,7 @@ import {
   StageInfo,
   StepInfo,
   StepLogBufferInfo,
+  TAIL_CONSOLE_LOG,
```

Comment:
LOG_FETCH_SIZE on line 8 needs removing now as its not used anymore

Reply:
Fixed in 86a4ea957792ae4e21db14e015a2025edc385b8a